Ds3/Runtime/Network.cs

```diff
@@ -57,6 +59,21 @@ internal class Network : INetwork
             this._readWriteTimeout = readWriteTimeout;
             this._requestTimeout = requestTimeout;
             this._connectionLimit = connectionLimit;
+
+            _createDs3WebRequestFunc = this.CreateDs3WebRequest;
+        }
+
+        public Network(
```

Comment:
I would make this internal.  We probably don't want to expose this in the external api.